cluster/gce/gci/configure.sh

```diff
@@ -393,6 +393,58 @@ function load-docker-images {
   fi
 }
 
+# If we are on ubuntu we can try to install docker
+function install-docker {
+  # bailout if we are not on ubuntu
+  if ! command -v apt-get >/dev/null 2>&1; then
+    echo "Unable to install automatically install docker. Bailing out..."
```

Comment:
if this happens we probably need to be using apt?
or we are on COS? does this change break COS?

Reply:
COS images already have docker. we only get here if `CONTAINER_RUNTIME` is set to `docker`. So if we have a COS image with docker, we have to fail fast!

Reply:
if CONTAINER_RUNTIME is docker && docker is not in path, seems fine :-)